src/conn/conn_api.c

```diff
@@ -1310,6 +1310,8 @@ __conn_open_session(WT_CONNECTION *wt_conn, WT_EVENT_HANDLER *event_handler, con
 
     session_ret = NULL;
     WT_ERR(__wt_open_session(conn, event_handler, config, true, &session_ret));
+    //set the open_session method's session name
```

Comment:
@y123456yz thanks for your patience. I think we can get rid of this comment.

Reply:
hi,  @Siddhartha8899 
I have delete the comment, thanks.

Reply:
Hi @y123456yz, perhaps you forgot to push the commit for the same.

Reply:
hi, @Siddhartha8899
sorry， I Submitted to error branch , I have delete the comment.

thanks.